keymaps/omnisharp-atom.cson

```diff
@@ -33,6 +33,7 @@
   'ctrl-alt-down': 'omnisharp-atom:navigate-down'
   'ctrl-.': 'omnisharp-atom:get-code-actions'
   'alt-enter': 'omnisharp-atom:get-code-actions'
+  'ctrl-shift-t': 'omnisharp-atom:find-symbols'
```

Comment:
You can also press `cmd-.` then a key to see if it's used. 

We should also add `ctrl-,` for non resharper users :)